### ISSUE TITLE
[Bug]: yaml: line 32: found character that cannot start any token

### ISSUE THREAD

**Issue by danielkrajnik** [LemmyNet/lemmy#4133]
### Requirements

- [X] Is this a bug report? For questions or discussions use https://lemmy.ml/c/lemmy_support
- [X] Did you check to see if this issue already exists?
- [X] Is this only a single bug? Do not put multiple bugs in one issue.
- [X] Is this a backend issue? Use the [lemmy-ui](https://github.com/LemmyNet/lemmy-ui) repo for UI / frontend issues.

### Summary

Follow all the steps in the official lemmy documentation to install the server via docker.

run `docker compose up -d`

error: `parsing /home/user/lemmy/docker-compose.yml: yaml: line 32: found character that cannot start any token`

line 32 is:
`  32   │ {% if lemmy_env_vars is defined and lemmy_env_vars|length > 0 %}`

### Steps to Reproduce

1.
2.
3.

### Technical Details

-

### Version

-

### Lemmy Instance URL

-

**Comment by danielkrajnik** [comment]
Wait... are these files just copy-pasted from ansible? Are they just dropped "raw" and users are expected to figure out how to edit them on our own?

**Comment by Nutomic** [comment]
Yes this could be clarified in the instructions. Or link to a specific commit for these files to avoid pulling in unexpected changes.
